libsolidity/codegen/mlir/Passes.cpp

```diff
@@ -209,10 +209,40 @@ static mlir::ModuleOp extractRuntimeModule(mlir::ModuleOp creationMod) {
   return {};
 }
 
+DEFINE_SIMPLE_CONVERSION_FUNCTIONS(llvm::TargetMachine, LLVMTargetMachineRef)
```

Comment:
@PavelKopyl Did I miss this in the llvm headers?

Reply:
As I understand it should be defined in `include/llvm/Support/CBindingWrapping.h`

Reply:
Sorry I was not clear. I meant if I missed the header that does `DEFINE_SIMPLE_CONVERSION_FUNCTIONS(llvm::TargetMachine, LLVMTargetMachineRef)`?
I guess not, as I can see the same in lib/ExecutionEngine/Orc/OrcV2CBindings.cpp:107